src/GameClientPlayer.cpp

```diff
@@ -800,13 +800,17 @@ void GameClientPlayer::AddJobWanted(const Job job, noRoadNode* workplace)
 
 void GameClientPlayer::JobNotWanted(noRoadNode* workplace,bool all)

```

Comment:
the "workplace" here should be const, since its not modified inside this function?